pyv/transpiler.py

```diff
@@ -25,6 +25,17 @@ def is_mutable(scopes, target: str) -> bool:
     return _is_mutable(scopes, target)
 
 
+def new_expr(source: str, node: ast.AST) -> ast.AST:
```

Comment:
Could move to `py2many/ast_helpers.py` which has similar code.

Reply:
I used create_ast_node in 301e11b4c04ad80b27e36e4c0d3ddd87a7b36aa2 instead.